lib/app.js

```diff
@@ -27,18 +29,23 @@ function run () {
     logger.error('[%d] Retrying in 10s...', count)
   }
 
-  const runner = () => runOnce()
-    .then(() => clearInterval(handler))
-    .catch(errorHandler)
+  const runner = () =>
+    settingsLoader.loadLatest()
+      .then((loadedSettings) => {
+        settings = loadedSettings
```

Comment:
I think we can delete this line, change ``loadedSettings`` to ``settings``, and just pass settings to ``loadSanctions`` and ``startServer``.

In order to do this, we need to change the code to something like this:

```javascript
    settingsLoader.loadLatest()
      .then(settings => {
        clearInterval(handler)
        return loadSanctions(settings)
        .then(() => startServer(settings))
        .catch(errorHandler)
      })
```